Lua error when editing empty file

If you're in an empty file and press backspace it gives this error:
```
[ERROR] addons/starfall/lua/starfall/editor.lua:327: attempt to perform arithmetic on a nil value
  1. unknown - addons/starfall/lua/starfall/editor.lua:327

Timer Failed! [validationTimer][@addons/starfall/lua/starfall/editor.lua (line 351)]
```

Editor doesn't break however.

--- a/lua/starfall/editor.lua
+++ b/lua/starfall/editor.lua
@@ -315,6 +315,7 @@ if CLIENT then
 
 		local function valid ()
 			local code = SF.Editor.getActiveTab().code
+			if code and code == "" then SF.Editor.runJS( "editor.session.clearAnnotations(); clearErrorLines()" ) return end
 
 			local err = CompileString( code, "Validation", false )
 

